contrib/opentimelineio_contrib/application_plugins/tests/test_rv_reader.py

```diff
@@ -46,7 +47,10 @@
 RV_OTIO_READER_VERSION = '1.0'
 
 RV_ROOT_DIR = os.getenv('OTIO_RV_ROOT_DIR', '')
-RV_BIN_DIR = os.path.join(RV_ROOT_DIR, 'bin')
+RV_BIN_DIR = os.path.join(
+    RV_ROOT_DIR,
+    'MacOS' if platform.system() == 'Darwin' else 'bin'
+)
```

Comment:
This fix should probably be in the main branch, right?

Reply:
Yes, there are a few problems with running the adaptor as-is on Mac.

Reply:
I've separated out the issues with the unit test into their own PRs with own Issues logged, so this PR contains only the changes relevant to bounds.  Other PRs here:
https://github.com/PixarAnimationStudios/OpenTimelineIO/pull/1147
https://github.com/PixarAnimationStudios/OpenTimelineIO/pull/1149